postgres-appliance/spilok8s.yaml

```diff
@@ -85,7 +85,8 @@ kind: Endpoints
 metadata:
   name: &cluster_name spilodemo
   labels:
-    app: *cluster_name
+    application: spilo
+    spilo-cluster: *cluster_name
```

Comment:
Because it is already 'spilo-cluster' in all other places. Moreover, we have 'spilo-role' label.